### DISCUSSION TITLE
Trying to remove song history and playlist from player.

### DISCUSSION THREAD

**Discussion by HairyToes** [AzuraCast/AzuraCast#7216] [General]
Hello,

New user here. I'm trying to remove song history and playlist from the web player on my public page. I have tried everything from this link with no luck: [https://github.com/AzuraCast/AzuraCast/issues/1866](https://github.com/AzuraCast/AzuraCast/issues/1866)

I've tried both the CSS and JS code. I have successfully removed a few items on the player with no problem. 

My code looks like this

```
.now-playing-art img {display:none !important; } .time-display-progress {display:none !important; }
.time-display-played {display:none; }
.time-display-progress {display:none !important; }
.progress-bar bg-secondary {display:none !important; } .time-display-total { display:none !important; }
.card-actions {display:none; }
```

What am I doing wrong?

Thank you.

**Comment by gAlleb** [comment]
```css
    .card-body.buttons {
        display: none;
    }
```

https://www.w3schools.com/css/

https://developer.chrome.com/docs/devtools/css

https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/how_to/examine_and_edit_css/index.html

https://developer.mozilla.org/en-US/docs/Web/CSS

**Reply by HairyToes** [reply]
Perfect !

Thank you gAlleb
